e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,206 @@
+      cy.cleanupProjectsByIDPrefix(adminToken, cypressPrefix);
+    });
+  });
+
+  beforeEach(() => {
+    cy.restoreStorage();
+  });
+
+  afterEach(() => {
+    cy.saveStorage();
+  });
+
+
+  it('list projects', () => {
+    cy.request({
+      auth: { bearer: adminToken },
+      method: 'GET',
+      url: '/apis/iam/v2beta/projects'
+    }).then((response) => {
+      expect(response.status).to.equal(200);
+      response.body.projects.map((project: Project) => project.id)
+        .forEach((id: string) => {
+        cy.get('chef-table chef-td').contains(id);
+      });
+    });
+  });
+
+  it('can create a project', () => {
+    cy.get('#create-button').contains('Create Project').click();
+    cy.get('app-create-object-modal chef-modal').should('have.class', 'visible');
+
+    // we increase the default delay to mimic the average human's typing speed
+    // only need this for input values upon which later test assertions depend
+    // ref: https://github.com/cypress-io/cypress/issues/534
+    cy.get('[data-cy=create-name]')
+      .type(projectName, { delay: 50 }).should('have.value', projectName);
+
+    cy.get('[data-cy=create-id]').should('not.be.visible');
+    cy.get('[data-cy=edit-button]').contains('Edit ID').click();
+    cy.get('[data-cy=id-label]').should('not.be.visible');
+    cy.get('#id-input').should('have.value', projectID);
+
+    cy.get('[data-cy=save-button]').click();
+    cy.get('app-create-object-modal chef-modal').should('not.be.visible');
+    cy.get('chef-notification.info').should('be.visible');
+    cy.contains(projectName).should('exist');
+    cy.contains(projectID).should('exist');
+
+    cy.url().should('include', `/settings/projects/${projectID}`);
+  });
+
+  it('can create a rule for the new project', () => {
+    cy.get('app-authorized button').contains('Create Rule').click();
+
+    cy.url().should('include', `/settings/projects/${projectID}/rules`);
+    cy.get('chef-page').should('be.visible');
+
+    cy.get('#create-name input').type(ruleName, { delay: 50 }).should('have.value', ruleName);
+    cy.get('[data-cy=edit-id]').click();
+
+    cy.get('#create-id input').should('have.value', ruleID);
+
+    // Verify correct attributes/operators are selectable for each resource
+    // Event
+    cy.get('#create-type-dropdown').select('Event');
+    cy.get('[data-cy=attribute-dropdown]').select('Chef Organization');
+    cy.get('[data-cy=attribute-dropdown]').select('Chef Server');
+    cy.get('[data-cy=operator-dropdown]').select('equals');
+    cy.get('[data-cy=operator-dropdown]').select('member of');
+
+    // Node
+    cy.get('#create-type-dropdown').select('Node');
+    const nodeAttributes = ['Chef Organization', 'Chef Server', 'Environment',
+        'Chef Role', 'Chef Tag', 'Chef Policy Name', 'Chef Policy Group'];
+
+    nodeAttributes.forEach((att: string) => {
+        cy.get('[data-cy=attribute-dropdown]').select(att);
+    });
+
+    cy.get('[data-cy=operator-dropdown]').select('member of');
```

Comment:
This line is not doing anything.

Reply:
We wanted to have the test select each available option to confirm they populate as expected.

Reply:
Oh, of course. That makes sense. 👍